homeassistant/components/sensor/velbus.py

```diff
@@ -0,0 +1,47 @@
+"""
+Velbus sensors.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/velbus/
```

Comment:
Url doesn't have correct format. Look at other platforms for the correct format.